GravityKit/GravityView
# Adds `url` input type for View settings fields, converts a few settings to use it

This takes advantage of browser validation (related to #2159), while making a very small change.

I considered updating the `GravityView_FieldType` abstract class to add an `input_type` and just use the existing `GravityView_FieldType_text` class, but I determined that this was cleaner.

## To test: 

In the View editor, test the Delete Entry Redirect URL, No Entries Redirect URL, and Edit Entry Redirect URL settings.

- Add URLs with special characters. Make sure they are saved and stored properly
- Test to make sure redirections are correct

💾 [Build file](https://www.dropbox.com/scl/fi/p941t5qjamoll7z7xgjzp/gravityview-2.31-cd843a70b.zip?rlkey=u38prlk7cu9a2cagg5gftlz2b&dl=1) (cd843a70b).
